OpenShift/common.sh

```diff
@@ -43,3 +43,31 @@ if [ ${#PULL_SECRET} = 0 ]; then
   echo "Get a valid pull secret (json string) from https://cloud.openshift.com/clusters/install#pull-secret"
   exit 1
 fi
+
+
+WORKING_DIR=${WORKING_DIR:-"/opt/install-scripts"}
```

Comment:
Nit: I'd say for install-scripts we should stick to the ~ folder to avoid spreading stuff in different folders

Reply:
Now based on SCRIPTDIR